src/parse_args.ts

```diff
@@ -33,6 +38,10 @@ program
   .option('-s, --suite-params <jsonstring>', 'Variables', '{}')
   .option('-e, --environment <envname>', 'e.g. production', 'development')
   .option('-j, --json', 'output newline delimited JSON')
```

Comment:
will remove `--json` in major. We can start using `--reporter json` from heartbeat